types/shippo/index.d.ts

```diff
@@ -1,65 +1,501 @@
-// Type definitions for shippo 1.5
+// Type definitions for shippo 1.7
 // Project: https://github.com/goshippo/shippo-node-client
 // Definitions by: Saiichi <https://github.com/saiichihashimoto>
+//                 James Lismore <https://github.com/jlismore>
```

Comment:
Just to confirm, are you looking to become an owner/review for PRs to these types, or just noting your contribution? If the latter, you shouldn't add this: https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
This was intentional - the package seems defacto unmaintained